docs/api/users-get-users-first-names.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+---
+
+# Get users first names
+
+Returns an array of  [`user`](user) objects that contains only the property specified by the `first_name` parameter, if it exists.
+
+## URL
+
+```shell
+
+{server_url}/users/:first_name
+```
```

Comment:
This syntax doesn't work for me when I test it.
Are you sure this is supported?

Reply:
I am not sure this is supported. I've spent the last half hour trying to find query parameters that allow a GET request to return just a single property but I can't figure it out. My understanding is that this kind of functionality would have to be built into the API, that it's not an inherent GET capability. So I'm not sure where to go from here. If I signed up for a reference topic that can't actually be described as supported in the API, should I find a different topic?